### PR TITLE
docs: add note for deploying to Fargate

### PR DESCRIPTION
**Pull request by wafuwafu13**
### Description

Add note for deploying to Fargate

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [ ] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: docs/deploy/installation.md [hunk 1]
```diff
@@ -191,7 +191,10 @@ We recommend using the Helm chart to install the controller. The chart supports
     helm install aws-load-balancer-controller eks/aws-load-balancer-controller -n kube-system --set clusterName=<cluster-name>
     ```
 
+    If you're deploying the controller to Amazon EC2 nodes that have [restricted access to the Amazon EC2 instance metadata service (IMDS)](https://aws.github.io/aws-eks-best-practices/security/docs/iam/#restrict-access-to-the-instance-profile-assigned-to-the-worker-node), or if you're deploying to Fargate, then add the following flags to the helm command that follows:
 
+    - `--set region=<region-code>`
+    - `--set vpcId=<vpc-id>`
```
Comment (wafuwafu13) [inline docs/deploy/installation.md:197]: If not set:
```
$ kubectl get pods --all-namespaces                                                
NAMESPACE     NAME                                            READY   STATUS             RESTARTS      AGE
kube-system   aws-load-balancer-controller-6c5d84f776-bcpfn   0/1     CrashLoopBackOff   4 (13s ago)   3m9s
kube-system   aws-load-balancer-controller-6c5d84f776-hkkp9   0/1     CrashLoopBackOff   4 (14s ago)   3m9s
...
```

File: docs/examples/echo_server.md [hunk 2]
```diff
@@ -86,6 +86,15 @@ In this walkthrough, you'll
...
 1.  Deploy all the echoserver resources (namespace, service, deployment)
 
+    If you're deploying to Fargate, then add the Fargate profile.
+
+    ```bash
+    eksctl create fargateprofile \
+    --cluster <cluster-name> \
+    --name <profile-name> \
+    --namespace echoserver
+    ```
```
Comment (wafuwafu13) [inline docs/examples/echo_server.md:96]: If not create:
```
$ kubectl describe pod echoserver-dc496b9-lqdcg -n echoserver
Events:

  Warning  FailedScheduling  2m11s  default-scheduler  0/4 nodes are available: 4 node(s) had untolerated taint {eks.amazonaws.com/compute-type : fargate}. preemption: 0/4 nodes are available: 4 Preemption is not helpful for scheduling.
```

Comment (oliviassss) [reply docs/examples/echo_server.md:96]: @wafuwafu13, sorry I added this comment long ago, but just realized I didn't public the comment.

**Comment by oliviassss** [comment]
@wafuwafu13, hey just to check if there is any plan to update this PR per the comment? Thanks

File: docs/examples/echo_server.md (hunk 2 above)
Comment (wafuwafu13) [reply docs/examples/echo_server.md]: Fargate profile is not the only setting for aws-load-balancer-controller, so deleted https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3322/commits/27d86d6970c555a1d47bf4226505d06d4a95a06a

**Pull request closed without merging**